Sources/StreamChat/Models/Message.swift

```diff
@@ -84,19 +84,35 @@ public struct _ChatMessage<ExtraData: ExtraDataTypes> {
     public let reactionScores: [MessageReactionType: Int]
     
     /// The user which is the author of the message.
-    public let author: _ChatUser<ExtraData.User>
+    ///
+    /// Author is loaded lazily on demand to maintain high performance, so this property is not thread safe
```

Comment:
Great that you put those comments there. Can we just tweak them a bit? Something like:
```
    /// The user who is the author of the message.
    ///
    /// - Important: The `author` property is loaded and evaluated lazily to maintain high performance.
    ///
```

There's no need to mention the potential thread issue, we will fix it shortly. It's just a temporary thing.